src/command_modules/azure-cli-batch/setup.py

```diff
@@ -34,6 +34,7 @@
 
 DEPENDENCIES = [
     'azure-batch==4.0.0',
+    'azure-batch-extensions>=1.0,<1.1',
```

Comment:
What's this for?
I guess it has something to do with this comment "Includes azure-namespace dependencies required by new extension package"?

Reply:
@derekbekoe - Yes this is our SDK extensions package. It uses the azure namespace when installing so we cannot add it as a dependency to the new CLI extension wheel, so we have to add it here instead in order for it to be installed correctly.

Reply:
Okay yes.